tpm

```diff
@@ -29,6 +29,7 @@ set_default_tpm_path() {
 
 	if [ -f "$xdg_tmux_path/tmux.conf" ]; then
 		tpm_path="$xdg_tmux_path/plugins/"
+		BINDINGS_DIR="XDG_CONFIG_HOME=$XDG_CONFIG_HOME $BINDINGS_DIR"
```

Comment:
I expected this PR to only handle tildae expansion in XDG_CONFIG_HOME.
The scope here expanded, no?

Reply:
Turned out tilde expansion didnt seem to work on my MacBook, then I noticed that tpm didn't seem to honor XDG_CONFIG_HOME for downloading plugins in the first place in my tests. But sure if this is to much of a change feel free to reject it

Reply:
I could do a separate one limited to the tilde expansion we talked about before